.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+sudo: required
+dist: trusty
+node_js: stable
```

Comment:
worth caching npm modules to speed things up a little?

https://docs.travis-ci.com/user/caching/#Configuration